src/modules/Channel/components/Message/index.tsx

```diff
@@ -51,7 +52,7 @@ const Message = (props: MessageProps) => {
         && (
           config?.groupChannel?.showSuggestedRepliesFor === 'all_messages'
             ? true
-            : message.messageId === currentGroupChannel?.lastMessage?.messageId
+            : message.messageId === allMessages[allMessages.length - 1].messageId
```

Comment:
It will be good if we comment why we don't use `lastMessage` here

Reply:
showSuggestedRepliesFor can have two values: all_messages, and last_message_only. if last_message_only, we render only if message is the last one.

Reply:
I'm sorry, I was talking about leaving a comment on the line 😅